Tasks/DownloadBuildDropV0/task.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "61F2A582-95AE-4948-B34D-A1B3C4F6A737",
+  "name": "DownloadBuildDrop",
+  "friendlyName": "Download Build Drop",
+  "description": "Download Build Drop",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 137,
+    "Patch": 0
+  },
+  "groups": [],
+  "demands": [],
+  "inputs": [
+	{
+      "name": "buildid",
+      "type": "string",
+      "label": "The specific build to download from",
+      "required": false,
+      "helpMarkDown": "The build to download from. Target the current build if left blank."
+    },
+	{
+      "name": "artifactname",
+      "type": "string",
+      "label": "The name of artifact to download.",
+      "required": true,
+      "helpMarkDown": "The name of artifact to download. The artifact must be a build drop."
+    },
+    {
+      "name": "targetpath",
+      "type": "filePath",
+      "label": "Path to download to",
+      "defaultValue": "$(Build.DropDownloadingDirectory)",
```

Comment:
why default to this variable? is this a predefined variable?

Reply:
removed